src/app/tab/tab.component.ts

```diff
@@ -225,4 +272,13 @@ export class TabComponent implements OnInit {
     return styles;
   }
 
+  setCovidCountryStyles(c: any): Object {
+    const styles = {
```

Comment:
In order to get the sizing correct based on my latest changes with positioning, this should match the way clocks are done. I think you only need to add justify-content and width here and you should be good.

```
  setTimeStyles(clock): Object {
    const styles = {
      'font-size' : this.shared.getFontSize(clock.scaling, 1.7),
      'justify-content' : this.getAlignment(clock.position),
      'width' : this.shared.getPercentWidth(clock.span),
      'order' : this.shared.getOrder(clock.id, clock.position),
      'transform' : this.shared.getOffset(clock.offset),
      'margin' : this.shared.getMargin(clock.marginHeight, clock.marginWidth)
    };
    return styles;
  }
```